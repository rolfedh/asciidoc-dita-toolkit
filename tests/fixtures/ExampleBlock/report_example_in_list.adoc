// Unsupported examples in list items:
* A list item.
+
.An example title
====
An unsupported example.
====

- A list item.
+

[example]
An unsupported example.

. A list item.
+
A paragraph.
+
====
An unsupported example.
====
