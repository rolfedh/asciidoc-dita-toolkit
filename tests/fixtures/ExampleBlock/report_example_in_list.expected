// Unsupported examples in list items:
* A list item.
+
//
// ADT ExampleBlock: Move this example block to the main body of the topic 
// (before the first section header) for DITA 1.3 compliance.
//
.An example title
====
An unsupported example.
====

- A list item.
+

//
// ADT ExampleBlock: Move this example block to the main body of the topic 
// (before the first section header) for DITA 1.3 compliance.
//
[example]
An unsupported example.

. A list item.
+
A paragraph.
+
//
// ADT ExampleBlock: Move this example block to the main body of the topic 
// (before the first section header) for DITA 1.3 compliance.
//
====
An unsupported example.
====
